packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/core/ovirt_internal.py

```diff
@@ -706,21 +706,30 @@ def _results(rc, out):
         ),
         condition=lambda self: (
             self.environment[oengcommcons.KeycloakEnv.ENABLE] and
-            not self.environment[oengcommcons.KeycloakEnv.CONFIGURED]
+            self.environment[osetupcons.CoreEnv.ACTION]
+                == osetupcons.Const.ACTION_SETUP
         )
     )
     def _closeup(self):
         self.dialog.note(
             text=_(
-                "Please use the user '{keycloakadmin}' and password specified in "
-                "order to login to Keycloak admin console\n"
-                "Please use the user '{user}' and password specified in "
-                "order to login using Keycloak SSO"
+                "To login to oVirt using Keycloak SSO, "
+                "enter '{user}' as username "
+                "and the password provided during the setup.\n"
```

Comment:
Perhaps, to match other texts, "during Setup\n" (no "the", capital 'S', no tailing dot. Not sure I like last one - perhaps we better add dots everywhere - requires more patches).

Reply:
Fixing "during Setup\n". Thanks!